MANIFEST.in

```diff
@@ -0,0 +1,8 @@
+graft src/python/grpcio/grpc
```

Comment:
Turns out that there's a way to rename the `MANIFEST.in` file; if I can find all the configuration settings required for the tools to be aware of the change I'll go with that.